experimental/linting_tool/lib/repository/api_provider.dart

```diff
@@ -5,14 +5,15 @@
 import 'dart:convert';
 import 'package:http/http.dart' as http;
 import 'package:linting_tool/model/rule.dart';
+import 'package:yaml/yaml.dart';
 
 class APIProvider {
   final _baseURL = 'https://dart-lang.github.io/linter';
   final http.Client httpClient;
   APIProvider(this.httpClient);
 
   Future<List<Rule>> getRulesList() async {
-    http.Response response =
+    var response =
```

Comment:
`final`? =)

Reply:
Sure.

Reply:
I wonder why we don't have lint rules enabled to pick this up? Should I create a new PR for this?

Reply:
I suspect we are missing a lint rule. What lints are setup for this project?

Reply:
I have used the default lints that are used in most of the projects in the repo.
[Example](https://github.com/flutter/samples/blob/master/veggieseasons/analysis_options.yaml)